github_stargazers/github_stargazers.py

```diff
@@ -119,8 +119,12 @@ def is_stargazer(self, user: str) -> bool:
 @click.argument('username_and_repository')
 @click.option('--user', default=None, help='User name to see if it is a stargazer')
 def process_command(username_and_repository, user):
-    github = GitHub(username_and_repository)
-
+    try:
+        github = GitHub(username_and_repository)
+    except UsernameRepositoryError:
+        message = "Wrong Arguments it should be of form username/repository, e.g marius92mc/github-stargazers"
```

Comment:
I am thinking if `"Wrong arguments. It should be of form username/repository, e. g. marius92mc/github-stargazers"` would be more clear to the user. What do you think?